specification/monitor/resource-manager/microsoft.insights/preview/2018-06-01-preview/guestDiagnosticSettingsAssociation_API.json

```diff
@@ -0,0 +1,444 @@
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/GuestDiagnosticSettingsAssociationResourcePatch"
+                        },
+                        "description": "Parameters supplied to the operation."
+                    }
+                ],
+                "responses": {
+                    "default": {
+                        "description": "Error response describing why the operation failed.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    },
+                    "200": {
+                        "description": "An existing guest diagnostics setting resource was successfully updated.",
+                        "schema": {
+                            "$ref": "#/definitions/GuestDiagnosticSettingsAssociationResource"
+                        }
+                    }
+                },
+                "x-ms-examples": {
+                  "Update a service diagnostic setting": { "$ref": "./examples/updateGuestDiagnosticSettingsAssociation.json" }
+                }
+              }
+        },
+        "/subscriptions/{subscriptionId}/providers/microsoft.insights/guestDiagnosticSettingsAssociations": {
+            "get": {
+                "description": "Get a list of all guest diagnostic settings association in a subscription.",
+                "operationId": "guestDiagnosticsSettingsAssociation_List",                
+                "x-ms-pageable": {
+                    "nextLinkName": null
+                },
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "The request succeeded.",
+                        "schema": {
+                            "$ref": "#/definitions/GuestDiagnosticSettingsAssociationList"
+                        }
+                    },
+                    "default": {
+                        "description": "An error occurred and the list of guest diagnostic settings association could not be retrieved.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/microsoft.insights/guestDiagnosticSettingsAssociations": {
+            "get": {
+                "description": "Get a list of all guest diagnostic settings association in a resource group.",
+                "operationId": "guestDiagnosticsSettingsAssociation_ListByRG",                
```

Comment:
Please revert this to "guestDiagnosticsSettingsAssociation_ListByResourceGroup"
This one didn't need to be changed.

Reply:
Done